.github/workflows/main.yml

```diff
@@ -6,7 +6,10 @@ on:
       - master
       - "[0-9]+.[0-9]+.x"
     tags:
-      - "*"
+      - "[0-9]+.[0-9]+.[0-9]+"
+      - "[0-9]+.[0-9]+.[0-9]rc[0-9]+"
+      - "[0-9]+.[0-9]+.[0-9]a[0-9]+"
```

Comment:
I think the two lines for `a` and `b` can be removed, as we don't intend to make this type of releases (at least for now). Having it here might suggest that we do, and adding them later in case we start to have alpha/beta releases is trivial.